.eslintignore

```diff
@@ -1,5 +0,0 @@
-.publish/*
```

Comment:
`semistandard` doesn't seem to read from this file. I added config to `package.json` instead.